package.json

```diff
@@ -29,15 +29,16 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@filipeop/notifications-panel": "^0.0.2",
     "@babel/core": "^7.18.6",
     "@babel/preset-env": "^7.18.6",
     "@babel/preset-react": "^7.18.6",
+    "@filipeop/notifications-panel": "^0.0.2",
```

Comment:
Are we still using this package?

Reply:
Yes, we ran into some difficulty having @filipeotero to publish under dynamods scope..